scripts/build/index.ts

```diff
@@ -110,18 +110,26 @@ export const addVersionLast = (feature: WalkOutput): void => {
 };
 
 /**
```

Comment:
Should we have `/* c8 ignore start */`  in line 112 instead of line 140?

Reply:
I'm not sure!  Do you think it would be good to test the data bundle creation?

Reply:
hm yes, I think you are right. I was just looking at the coverage report and saw these lines still uncovered. Do you want to write some tests for this?

Reply:
I'd be down, but would it be alright to bump those changes to the next PR instead?  (Currently unable to do so right now, happy to explain why in a DM!)